cmd/root.go

```diff
@@ -47,25 +47,13 @@ func Prepare() *cobra.Command {
 	viper.SetEnvPrefix("PGROLL")
 	viper.AutomaticEnv()
 
-	rootCmd.PersistentFlags().String("postgres-url", "postgres://postgres:postgres@localhost?sslmode=disable", "Postgres URL")
-	rootCmd.PersistentFlags().String("schema", "public", "Postgres schema to use for the migration")
-	rootCmd.PersistentFlags().String("pgroll-schema", "pgroll", "Postgres schema to use for pgroll internal state")
-	rootCmd.PersistentFlags().Int("lock-timeout", 500, "Postgres lock timeout in milliseconds for pgroll DDL operations")
-	rootCmd.PersistentFlags().String("role", "", "Optional postgres role to set when executing migrations")
-
-	viper.BindPFlag("PG_URL", rootCmd.PersistentFlags().Lookup("postgres-url"))
-	viper.BindPFlag("SCHEMA", rootCmd.PersistentFlags().Lookup("schema"))
-	viper.BindPFlag("STATE_SCHEMA", rootCmd.PersistentFlags().Lookup("pgroll-schema"))
-	viper.BindPFlag("LOCK_TIMEOUT", rootCmd.PersistentFlags().Lookup("lock-timeout"))
-	viper.BindPFlag("ROLE", rootCmd.PersistentFlags().Lookup("role"))
```

Comment:
This removes the ability to set these flags via env vars, which I don't think we want.